### PR TITLE
Add script to generate files after crd.yml.j2 changes

### PR DESCRIPTION
**Pull request by rooftopcellist**
This script may be useful for other people making changes to template files.  This will help keep people from editing the generated files manually when variables/attributes are added.  

cc @tchellomello

### CODE REVIEW

**Comment by rooftopcellist** [comment]
I also added a step top generate the bundle manifest to the `build.sh`.

**Review by shanemcd — CHANGES_REQUESTED** [review]
-1 on adding a the shell script at the root of the repo. Please put it in `scripts`.

**Comment by rooftopcellist** [comment]
woops, I meant to put that in the scripts directory, thanks for catching that, it is updated.

File: scripts/build.sh [hunk 1]
```diff
@@ -78,6 +78,7 @@ build_operator_image() {
 
 build_bundle_image() {
   echo "Building and pushing $BUNDLE_IMAGE image"
+  operator-sdk generate bundle --operator-name awx-operator --version $TAG
```
Comment (tchellomello) [inline scripts/build.sh:81]: That is great so we can have the option to generate the bundle here as well!

**Review by tchellomello — APPROVED** [review]
:fire: sweet!!

File: scripts/build.sh (hunk 1 above)
Comment (rooftopcellist) [reply scripts/build.sh:81]: One less step when testing changes that affect the bundle :smile:

**Pull request merged**